test262/main.js

```diff
@@ -140,7 +202,12 @@
     return { charts };
   }
 
-  function initializeChart(element, { datasets, metadata }) {
+  function initializeChart(
+    element,
+    { datasets, metadata },
+    xAxisTitle = "Time",
+    yAxisTitle = "Number of tests"
```

Comment:
I like using objects for options like these, to simulate names arguments:

```suggestion
    { xAxisTitle = "Time", yAxisTitle = "Number of tests" }
```

(prettier might not agree with this formatting)